keptn-gateway/cmd/api/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"fmt"
+	"keptn.sh/keptn-gateway/internal/gateway"
+)
+
+func main() {
+	gw := gateway.NewGateway()
+
+	err := gw.ListenAndServe()
+	if err != nil {
+		panic(fmt.Sprintf("cannot start server: %s", err))
+	}
```

Comment:
I suggest also handling SIGTERM here calling the shutdown method on the server if we receive from K8s the signal